### PR TITLE
 feat: 🚀 增加 prettier 插件入口 cjs文件

### LINKED ISSUE #127: 实现 prettier 插件开发

### PR DESCRIPTION
**Pull request by yuhao423**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/xun082/create-neat/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[√ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

- Closes #127
- Related to #24 

## What is the new behavior?

增加 prettier 插件入口 cjs文件

## Does this PR introduce a breaking change?

```
[ ] No
```

### CODE REVIEW

**Review by xun082 — APPROVED** [review]

File: packages/@plugin/plugin-prettier/index.cjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+const PluginConfig = require('./generator/index.cjs')
+
+
+const pluginPrettier = (buildTool)=>{
+
+
```
Comment (uaenaTzx) [inline packages/@plugin/plugin-prettier/index.cjs:7]: 无用的空行

**Comment by uaenaTzx** [comment]
prettier 对依赖的影响也要有

**Review by xun082 — APPROVED** [review]

File: packages/@plugin/plugin-prettier/generator/index.cjs [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+        useTabs: false,             // 使用空格代替制表符进行缩进
+        semi: true,                 // 在语句末尾添加分号
+        singleQuote: true,          // 使用单引号代替双引号
+        trailingComma: "es5",       // 尾随逗号的使用方式（es5 对象、数组等使用）
+        bracketSpacing: true,       // 对象文字中的括号间是否加空格
+        arrowParens: "always",      // 箭头函数参数使用圆括号
+        endOfLine: "lf"             // 换行符使用 lf (Unix 风格)    
     },
     devDependencies: {
       "@types/prettier": "^3.0.0",
```
Comment (uaenaTzx) [inline packages/@plugin/plugin-prettier/generator/index.cjs:15]: 这个不是必要依赖，得考虑后续在依赖处理中间件再做

Comment (yuhao423) [reply packages/@plugin/plugin-prettier/generator/index.cjs:15]: 那你说的这些直接删了吗

Comment (uaenaTzx) [reply packages/@plugin/plugin-prettier/generator/index.cjs:15]: 我认为 types 的包目前不用加吧，这个群里也聊过的，后面设计一个更好的方案来

**Pull request merged**